spec/services/teacher_training_public_api/assign_site_attributes_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe TeacherTrainingPublicAPI::AssignSiteAttributes do
+  let(:site) { described_class.new(site_from_api, provider).call }
+
+  let(:provider) { create(:provider) }
+  let(:site_from_api) do
+    Hashie::Mash.new(
```

Comment:
I think using this these days isn't recommended and we aren't in this codebase, could we maybe use the TTAPI helpers or some form of struct instead, also 👍 for the test data, its time for an early dinner 🧆 !